AzureDevOps.md

```diff
@@ -0,0 +1,101 @@
+# Azure DevOps and Titus
+
+How to create your pipelines using Azure and AzureDevOps
+
+### Requirements:
+- Azure account, a Resource Group and Owner permissions for it.
+- Azure CLI client installed
+- Terraform 0.15 or older
+
+## INSTRUCTIONS:
+
+Do "az login" and use your credentials to make sure you'll be able to run Terraform against your Resource Group.
+
+Sometimes the az login takes some time to finish, wait for the output showing the account details (containing tenantId, ...)
+
+### TERRAFORM - Create the required infrastructure
+
+Go to infra > azure > terraform
+
+For the first run, since there is no container registry, make sure backend, frontend and db-manager tf files are using the helloworld image.
+
+```bash
+    #name   = "titus-backend-${random_string.suffix.result}"
+    image  = "microsoft/aci-helloworld:latest"
+```
+
+Change the TF variables to use your own resource group id, name and location and Apply the TF plan.
+
+```bash
+# Get this information from your Azure account manager
+resource_group_id = "7d149fba-cc81-4bab-8461-ea09919d78cd"
```

Comment:
Shouldnt commit this - as its a template.

Reply:
Masked the value and documentation moved and formatted to match GCP docs